src/meta/src/barrier/worker.rs

```diff
@@ -490,6 +490,8 @@ impl<C: GlobalBarrierWorkerContext> GlobalBarrierWorker<C> {
 
             let reason = RecoveryReason::Failover(err.clone());
 
+            // We don't have span in release mode, so add an additional log here.
```

Comment:
This is not true. 🤔 Spans are only disabled on CI.

Reply:
You are right. I found the tracing span on another customer's cluster. 🤔